kentik_api_library/kentik_api/throttling/background_queue.py

```diff
@@ -1,3 +1,4 @@
+# mypy: ignore-errors
```

Comment:
What are the errors here? Mypy is already running on CI, so there shouldn't be too many issues here